.github/workflows/frontend-build.yml

```diff
@@ -0,0 +1,19 @@
+name: test frontend build
```

Comment:
you name it test but you don't run any test. You are building the frontend

Reply:
true dat - changes I will make 🦑